internal/querycoordv2/task/utils.go

```diff
@@ -180,14 +180,6 @@ func packReleaseSegmentRequest(task *SegmentTask, action *SegmentAction) *queryp
 	}
 }
 
-func packLoadMeta(loadType querypb.LoadType, collectionID int64, partitions ...int64) *querypb.LoadMetaInfo {
```

Comment:
NIT: it's recommended to keep this utility func in case of add/removal load meta attribute and make sure all occurrences are changed

Reply:
> If use the util function, I will create a function like func(arg1, arg2, arg3, arg4, arg5...), it's hard to maintain it if adding new args to it.

`ReplicaForPlan` cannot handle the problem, I will use `Replica` to replice it. wait for Resource Group Enhancement to handle it.